pints/toy/_rosenbrock.py

```diff
@@ -65,5 +70,54 @@ def optimum(self):
         return self._f.optimum()
 
     def __call__(self, x):
-        f = self._f(x)
-        return float('inf') if f == 0 else -np.log(f)
+        f = (1.0 + self._f(x))
```

Comment:
Can now avoid the variable `f` altogether!

Reply:
@MichaelClerx Cool done!